java/jar/jar.go

```diff
@@ -559,18 +559,18 @@ func (i *Info) parseProperties(ctx context.Context, r io.Reader) error {
 	var group, artifact, version string
 	s := bufio.NewScanner(r)
 	for s.Scan() && (group == "" || artifact == "" || version == "") {
-		b := bytes.TrimSpace(s.Bytes())
-		ls := bytes.SplitN(b, []byte("="), 2)
-		if len(ls) != 2 {
+		line := strings.TrimSpace(s.Text())
+		key, value, found := strings.Cut(line, "=")
```

Comment:
I wonder if we can rely on this always being `key=value` or if we should handle spaces?

Reply:
I think _technically_ spaces could be there: https://en.wikipedia.org/wiki/.properties
But I think that in practice, maven never emits spaces there.

Reply:
https://maven.apache.org/shared/maven-archiver/#pom-properties-content tells me it does not use spaces, so that's nice assurance